Animations/Fade.js

```diff
@@ -1,18 +1,18 @@
 var makeAnimation = require('../src/TuxAnimations');
```

Comment:
Add 'use strict'; on line one and leave a newline between that and variable declarations.